leapp/actors/__init__.py

```diff
@@ -92,36 +100,102 @@ def common_files_paths(self):
         """ Returns all common repository file paths. """
         return os.getenv("LEAPP_COMMON_FILES", "").split(":")
 
-    def get_folder_path(self, name):
+    def _get_folder_path(self, directories, name):
```

Comment:
why we are mixing `folder` and `directory` (not only here)? I prefer to use *directory* equivalent everywhere. So `_get_dir_path`, ....